userland/init/init.c

```diff
@@ -95,6 +101,8 @@ int main() {
                     reboot();
                 } else if (strncmp(readline, "uptime", 6) == 0) {
                     uptime();
+                } else if (strncmp(readline, "overflow", 8) == 0) {
```

Comment:
We should probably document this command. It needs an entry in `commands[]` and `NB_DOCUMENTED_COMMANDS` should be incremented.

Reply:
Will do!

Reply:
Done